swiftchat.go

```diff
@@ -157,13 +127,11 @@ func main() {
 
 	router.HandleFunc("/api/rooms/{RoomID:[0-9]+}", GetRoom).Methods("GET")
 
-	// TODO(pasha): Implement message create logic. Refactoring map with messages.
 	router.HandleFunc("/api/messages", CreateMessage).Methods("POST")
```

Comment:
you still have not removed obsolate route, you should use `/api/rooms/{RoomID:[0-9]+}/messages` not only for getting messages of specific room, but to create messages as well